packages/esm-patient-vitals-app/src/vitals/vitals-overview.scss

```diff
@@ -9,7 +10,7 @@
 .divider {
   width: 1px;
   height: layout.$spacing-05;
-  color: colors.$gray-20;
+  color: $ui-05;
```

Comment:
And here

Reply:
When imported as *, it should function properly without requiring the `vars` prefix.